src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -214,6 +213,7 @@ private void configChangeHandler(WhatHappened whatHappened, Node node) {
     @Override
     protected void startup() throws InterruptedException {
         context.get(CertificateManager.class).startMonitors();
+        subscribeToConfigChanges();
```

Comment:
subscribe registers a config change callback. That callback needs to be saved as a final field so that we don't duplicate callbacks